packages/lib/src/prod/remote-production.ts

```diff
@@ -172,7 +172,7 @@ export {__federation_method_ensure, __federation_method_getRemote , __federation
 
         if (id === '\0virtual:__federation_lib_semver') {
           const federationId = (
-            await this.resolve('@originjs/vite-plugin-federation')
+            await this.resolve('vite-plugin-federation/packages/lib')
```

Comment:
what does this change do?

Reply:
The forked version of the module is the entire repo v. the published npm module which just includes files at `dist`. Running the build command in the forked repo writes the files to `vite-plugin-federation/packages/lib/dist` and this change ensures that the consumer is able to find the `dist` folder